src/components/WorkOrders/JobView.js

```diff
@@ -15,7 +20,13 @@ const JobView = ({ pageNumber }) => {
     setError(null)
 
     try {
-      const data = await getRepairs(pageNumber)
+      var data = []
+
+      if (user.hasContractorPermissions) {
+        data = await getRepairs(pageNumber)
+      } else if (user.hasContractManagerPermissions) {
+        data = await getPendingApprovalRepairs(pageNumber)
+      }
```

Comment:
Probs should use `let` for ES6 syntax

Could you not do `const data = user.hasContractorPermissions ? await getRepairs(pageNumber) : await getPendingApprovalRepairs(pageNumber)`

Reply:
Sure happy to use `let` so that it follows ES6 syntax.

In addition, since we also have `hasAgentPermissions` I thought it's better to be more strict.